app/controller/List.js

```diff
@@ -120,13 +120,23 @@ Ext.define('WhatsFresh.controller.List', {
 		// then we check to see if a product is chosen, if one is we sort by product
 		console.log('In controller(home): Drop Down list Products');
 		// console.log(record);
+		var key = 0;
+		var ProdStore = Ext.getStore('Product');
+		if(key === 0){
+			ProdStore.insert(0, [
+				{
+					name: 'Please choose a product',
+					id: 0
+				}
+			]);
+			key = 1;
+		}
```

Comment:
This injection occurs in the Product store's callback - I will submit an edit this morning.